packages/web-extension/src/walletManager/types.ts

```diff
@@ -10,7 +10,7 @@ export enum WalletType {
 }
 
 /** For BIP-32 wallets: hash of extended account public key. For script wallets: script hash */
```

Comment:
Is the comment still relevant?

Reply:
I think yes, but it can be a different kind of hash. If wanted to make this an OpaqueString then I think it should be a WalletId rather than hash type.

Reply:
Yes the comment explains what the contents of wallet id could be, in the end we decided to just make it a string, should we change this?

Reply:
No need to change it. Thank you